frontend/src/pages/Partners/Partners.tsx

```diff
@@ -0,0 +1,225 @@
+import BackToHomeButton from '@components/BackToHomeButton/BackToHomeButton';
+import Container from '@components/Container/Container';
+import { H2, LinkToExtPage } from '@components/Typography';
+import BackButton from '@pages/BackButton/BackButton';
+import { routes } from '@routes';
+import router from 'next/router';
+import { MainTitle, Paragraph, MainLink, Link, Logo, Section, Logowrapper, Img } from './styled';
+
+import ceskoDigitalLogo from '../../pages/HomePage/PartnersSection/cesko-digital.png';
+import laboratorVodafoneLogo from '../../pages/HomePage/PartnersSection/laborator-vodafone.png';
+import nadaceCsLogo from '../../pages/HomePage/PartnersSection/nadace-cs.png';
+import blizKSobeLogo from './bliz-k-sobe.png';
+import zemeKvetLogo from './zemekvet.png';
+import mhmpLogo from './praha-logo.png';
+import otevrenoLogo from './otevreno.png';
+import ucitelNazivoLogo from './ucitel-nazivo.png';
+import ucitelskaPlatformaLogo from './ucitelska-platforma.png';
+import skavLogo from './skav.png';
+import eduLogo from './edu.png';
+import nadaceKarlaJaneckaLogo from './nadace-karla-janecka.png';
+import euLogo from './european-social-fund.png';
+import Meta from '@components/Meta/Meta';
+
+const links = [
+  { cd: { logo: ceskoDigitalLogo, href: 'https://cesko.digital/' } },
```

Comment:
je divný mít pole, kde jsou objekty, který mají jenom jeden klíč.

Změnil bych to buď na pole tohodle typu
```tsx
const links = [
  { name: 'cd', logo: ceskoDigitalLogo, href: 'https://cesko.digital/' },
  ...
]
```

a nebo 
```tsx
const links = {
  cd: { logo: ceskoDigitalLogo, href: 'https://cesko.digital/' },
  ...
}
```